src/features/__tests__/ScheduleCardCourses.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { fireEvent, waitForElement, cleanup } from '@testing-library/react';
+import { render } from '../../util/test-utils';
+import { ScheduleCardCourses } from '../schedule/ScheduleCardCourses';
+
+const mockCourse = {
```

Comment:
Let's build the tests to make use of the existing mock data at `src/api/student/__mocks__/courses.data.ts`